### PR TITLE
Update LDAP docs, add daedalus + icarus docs too

### PR DESCRIPTION
**Pull request by m1cr0man**
@butlerx review aswell please

### CODE REVIEW

File: docs/hardware/daedalus_icarus.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Daedalus' IP is `0.50`, Icarus' is `0.150`
+- Their iDRAC IPs are `1.50` and `1.150` respectively
+- Both run 802.3ad bonding on NICs 1 and 2, and have their iDRACs
+set up in "shared" mode.
+- Both boot from a 2x68gb SAS Hardware RAID1 pair, and also have
+3x600gb SAS used for ZFS + in the future Gluster.
+
+## Services
+
+- [LDAP](/services/ldap)
```
Comment (butlerx) [inline docs/hardware/daedalus_icarus.md:19]: Was git not migrated to these guys?

File: docs/services/ldap.md [hunk 2]
```diff
@@ -1,212 +1,223 @@
...
-dn: uid=username,ou=accounts,o=redbrick
-changetype: modify
-replace: loginShell
-loginShell: /usr/local/shells/disusered
+# Find all expired users. Notice here that you can query by hidden fields, but you can't read them
+ldapsearch -b o=redbrick -xLLL 'yearsPaid < 1' uid
 ```
 
-Where: username is who you're editing
+### ldapmodify recipes
```
Comment (butlerx) [inline docs/services/ldap.md:66]: Moving away from useradm and rb-ldap to ldapmodify. is a great idea.
Be aware that when creating users there's info that should be pulled from dcu ldap to seed accounts in redbrick. As well as creating home and webtree. And migrating type to associate. 
Tldr some tasks are to complicated for ldapmodify

File: docs/hardware/daedalus_icarus.md (hunk 1 above)
Comment (butlerx) [reply docs/hardware/daedalus_icarus.md:19]: Ignore i see where git runs

File: docs/services/ldap.md (hunk 2 above)
Comment (theycallmemac) [reply docs/services/ldap.md:66]: I would agree with this sentiment, it is definitely worth highlighting however I don't think this file requires any further changes at the moment.

**Review by theycallmemac — APPROVED** [review]

File: docs/services/ldap.md (hunk 2 above)
Comment (m1cr0man) [reply docs/services/ldap.md:66]: Yeah I think I will write an entirely separate document for adding new users.

Comment (butlerx) [reply docs/services/ldap.md:66]: could we create an issue for adding docs for adding new users, converting members to associates, and disabling a user

**Review by butlerx — APPROVED** [review]
